project-testnet.yaml

```
@@ -157,45 +159,40 @@ dataSources:
               - Transfer(address indexed from, address indexed to, uint256 amount)
   - <<: *ethereum
     options:
-      abi: queryRegistry
-      address: '0xabD5f22bBFcd48D913DFC0C1B6Cb08f38C8d2370'
+      abi: projectRegistry
+      address: '0xdD7bbd8bF59D11cac793389839fdcFD1468a0545'
     mapping:
       file: ./dist/index.js
       handlers:
-        - handler: handleNewQuery
+        - handler: handleNewProject
           kind: ethereum/LogHandler
           filter:
             topics:
-              - CreateQuery(uint256 queryId, address creator, bytes32 metadata, bytes32 deploymentId, bytes32 version)
-        - handler: handleUpdateQueryMetadata
+              - event CreateProject(address indexed creator, uint256 indexed projectId, string projectMetadata, ProjectType projectType, bytes32 deploymentId, bytes32 deploymentMetadata)
```

enum in event can not be correctly encoded, use `uint8` instead
```suggestion
              - event CreateProject(address indexed creator, uint256 indexed projectId, string projectMetadata, uint8 projectType, bytes32 deploymentId, bytes32 deploymentMetadata)
```